test/browser_test.rb

```diff
@@ -65,6 +65,15 @@ class BrowserTest < Minitest::Test
     assert_equal "4", @browser.version
   end
 
+  test "detects chrome browser" do
+    @browser.ua = $ua["CHROME"]
+    assert @browser.chrome?
+
+    @browser.ua = 'AnyOtherBrowser'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.